lib/src/Pos/Arbitrary/Block/Message.hs

```diff
@@ -0,0 +1,46 @@
+
+module Pos.Arbitrary.Block.Message
```

Comment:
Current convention is to define `Arbitrary` instances in the same package where types are defined. AFAIU, this module can be moved back to `block`. If that's the case, can you do it please?

Reply:
Sure.